From 44731d11a42f75f47e689a9004082b2ed42c6136 Mon Sep 17 00:00:00 2001
From: wusel <wusel+src@uu.org>
Date: Thu, 20 Jun 2024 00:26:28 +0200
Subject: [PATCH] Add Linksys EA8300 & MR8300

---
 targets/ipq40xx-generic | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/targets/ipq40xx-generic b/targets/ipq40xx-generic
index 83b4a128..739bfcb8 100644
--- a/targets/ipq40xx-generic
+++ b/targets/ipq40xx-generic
@@ -95,6 +95,26 @@ device('gl.inet-gl-b1300', 'glinet_gl-b1300', {
 
 device('linksys-ea6350v3', 'linksys_ea6350v3')
 
+device('linksys-ea8300-dallas', 'linksys_ea8300', {
+	packages = { -- Note: Gluon does not support third radio
+	'-ath10k-firmware-qca9888-ct',
+	'-ath10k-board-qca9888',
+	},
+	-- broken = true,
+	-- 3rd WiFi radio not supported (5GHz DFS channels 100+)
+	-- outdoor mode is broken due to 5GHz being limited to channels 36-64
+})
+
+device('linksys-mr8300-dallas', 'linksys_mr8300', {
+	packages = { -- Note: Gluon does not support third radio
+	'-ath10k-firmware-qca9888-ct',
+	'-ath10k-board-qca9888',
+	},
+	-- broken = true,
+	-- 3rd WiFi radio not supported (5GHz DFS channels 100+)
+	-- outdoor mode is broken due to 5GHz being limited to channels 36-64
+})
+
 
 -- NETGEAR
 
-- 
2.30.2

